2023-01-20 00:03:43 ERROR Element not present after 0.5 seconds on page: https://capital.com/
2023-01-20 00:03:43 ERROR 
Traceback (most recent call last):
  File "/home/runner/work/TestCapitalCom/TestCapitalCom/pages/base_page.py", line 75, in inner_function
    return func(*args, **kwargs)
  File "/home/runner/work/TestCapitalCom/TestCapitalCom/pages/base_page.py", line 316, in element_is_visible
    return Wait(self.browser, timeout).until(
  File "/home/runner/.cache/pypoetry/virtualenvs/testcapitalcom-G_ZVEfM_-py3.10/lib/python3.10/site-packages/selenium/webdriver/support/wait.py", line 95, in until
    raise TimeoutException(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: 
Stacktrace:
#0 0x555efeaf7303 <unknown>
#1 0x555efe8cbd37 <unknown>
#2 0x555efe9085b2 <unknown>
#3 0x555efe9086c1 <unknown>
#4 0x555efe942b34 <unknown>
#5 0x555efe9289ad <unknown>
#6 0x555efe94088c <unknown>
#7 0x555efe928753 <unknown>
#8 0x555efe8fba14 <unknown>
#9 0x555efe8fcb7e <unknown>
#10 0x555efeb4632e <unknown>
#11 0x555efeb49c0e <unknown>
#12 0x555efeb2c610 <unknown>
#13 0x555efeb4ac23 <unknown>
#14 0x555efeb1e545 <unknown>
#15 0x555efeb6b6a8 <unknown>
#16 0x555efeb6b836 <unknown>
#17 0x555efeb86d13 <unknown>
#18 0x7f27e519db43 <unknown>